swagger2locustio/generators/base_generator.py

```diff
@@ -1,19 +1,25 @@
+"""Module: Base Generator"""
+
 import logging
 from itertools import combinations
 from copy import deepcopy
 from typing import List, Dict, Union
 
 from swagger2locustio.templates import locustfile_templates as l_templates
 
-log = logging.getLogger(__name__)
+LOG = logging.getLogger(__name__)
 
 
 class BaseGenerator:
+    """Class: Base Generator"""
+
     def __init__(self, strict_level: int):
         self.strict_level = strict_level
         self.vars_without_values = {}
 
     def generate_locustfile(self, swagger_data: dict) -> str:
+        """Function: generate locustfile"""
```

Comment:
it seems to me that """Method: generate locustfile""" would be better

Reply:
Renamed: https://github.com/vsvirsky/swagger2locustio/commit/7abf54786a6c6c2bc1cab6ca78148862ccdb3756